src/player-service/playerRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { getPlayerById, getPlayersByTeam, placePlayerOnTransferList, transferPlayer } from './playerController';
+import { authMiddleware } from '../libs/common/middleware/auth';
+
+const router = Router();
+
+router.use(authMiddleware);
+
+router.get('/:teamId', getPlayersByTeam);
+
+router.post('/transfer-list', placePlayerOnTransferList);
```

Comment:
why the transfer is part of the player module?

Reply:
i think it makes sense here, since the way i see it is transfering is an action done on players, wdyt?